src/auth/azAuthentication.ts

```diff
@@ -17,7 +16,7 @@ export async function getAccessToken(): Promise<string> {
 export async function getAADToken(): Promise<string> {
   azPath = await io.which("az", true);
   return await getTokenFromAzCLI(
-    "account get-access-token --resource=" + resource
+    "account get-access-token --resource=" + await ManagementUrlHelper.getBaseUrl()
```

Comment:
For better readability, store in a variable and use.

Reply:
Suggest the same for other places too.

Reply:
File removed. Other places, variable used before adding to string